src/coreclr/jit/lsraxarch.cpp

```diff
@@ -2500,7 +2499,7 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
 
                 if (op3 != nullptr)
                 {
-                    srcCount += isRMW ? BuildDelayFreeUses(op3) : BuildOperandUses(op3);
+                    srcCount += isRMW ? BuildDelayFreeUses(op3, op1) : BuildOperandUses(op3);
```

Comment:
It looks like this covers the last few (last 5/20) usages in lsraxarch that weren't passing an RMW op to `BuildDelayFreeUses`, is that correct?
* It looks like most nodes get this covered by `BuildRMWUses` and only a handful of special nodes, like HWIntrinsics, are doing it "manually"

Reply:
That's right.